<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1006</width>
    <height>574</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>824</width>
         <height>458</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1006</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionCreate"/>
    <addaction name="actionConfig"/>
    <addaction name="actionRelaunch"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAboutQt"/>
    <addaction name="actionUpdateObject"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="debugDockWidget">
   <property name="allowedAreas">
    <set>Qt::AllDockWidgetAreas</set>
   </property>
   <property name="windowTitle">
    <string>Output</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2"/>
  </widget>
  <widget class="QDockWidget" name="objectDockWidget">
   <property name="windowTitle">
    <string>Object</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3"/>
  </widget>
  <widget class="QDockWidget" name="itemDockWidget">
   <property name="windowTitle">
    <string>Items</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_4"/>
  </widget>
  <action name="actionNew">
   <property name="text">
    <string>New Project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open Project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionCreate">
   <property name="text">
    <string>Create Projet Shortcut</string>
   </property>
  </action>
  <action name="actionConfig">
   <property name="text">
    <string>Project Config</string>
   </property>
  </action>
  <action name="actionRelaunch">
   <property name="text">
    <string>Relaunch</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionIPhone_3Gs_320x480">
   <property name="text">
    <string>iPhone 3Gs (320x480)</string>
   </property>
  </action>
  <action name="actionIPhone_4_640x960">
   <property name="text">
    <string>iPhone 4 (640x960)</string>
   </property>
  </action>
  <action name="actionIPhone_5_640x1136">
   <property name="text">
    <string>iPhone 5 (640x1136)</string>
   </property>
  </action>
  <action name="actionIPad_768x1024">
   <property name="text">
    <string>iPad (768x1024)</string>
   </property>
  </action>
  <action name="actionIPad_Retina_1536x2048">
   <property name="text">
    <string>iPad Retina (1536x2048)</string>
   </property>
  </action>
  <action name="actionAndroid_480x800">
   <property name="text">
    <string>Android (480x800)</string>
   </property>
  </action>
  <action name="actionAndroid_480x854">
   <property name="text">
    <string>Android (480x854)</string>
   </property>
  </action>
  <action name="actionAndroid_540x960">
   <property name="text">
    <string>Android (540x960)</string>
   </property>
  </action>
  <action name="actionAndroid_600x1024">
   <property name="text">
    <string>Android (600x1024)</string>
   </property>
  </action>
  <action name="actionAndroid_800x1280">
   <property name="text">
    <string>Android (720x1280)</string>
   </property>
  </action>
  <action name="actionAndroid_800x1280_2">
   <property name="text">
    <string>Android (800x1280)</string>
   </property>
  </action>
  <action name="actionAndroid_1080x1920">
   <property name="text">
    <string>Android (1080x1920)</string>
   </property>
  </action>
  <action name="actionPortrait">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Portrait</string>
   </property>
  </action>
  <action name="actionLandscape">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Landscape</string>
   </property>
  </action>
  <action name="actionReset_Zoom_100">
   <property name="text">
    <string>Reset Zoom (100%)</string>
   </property>
  </action>
  <action name="actionZoom_Out_50">
   <property name="text">
    <string>Zoom Out (50%)</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="text">
    <string>AboutQt</string>
   </property>
  </action>
  <action name="actionUpdateObject">
   <property name="text">
    <string>updateObject</string>
   </property>
   <property name="shortcut">
    <string>F6</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
